openapiart/generator.py

```diff
@@ -905,6 +905,33 @@ def _write_openapi_object(self, ref, choice_method_name=None):
                 self._write(2, "else:")
                 self._write(3, "self.choice = choice")
 
+            # write def set(self)
```

Comment:
I guess we should have a method which would write the set method `_write_set_method`.
It would help in maintainability and doing further modifications.
for example use have method to write `add` and `factory`:
```python
def _write_add_method:

def _write_factory_method:
```

Reply:
@dipendughosh agreed, let's refactor this out to a different method

Reply:
Created a new _write_get_method